homeassistant/components/simplisafe/__init__.py

```diff
@@ -296,6 +296,13 @@ def _async_register_base_station(
 ) -> None:
     """Register a new bridge."""
     device_registry = dr.async_get(hass)
+
+    # Check for an old system ID format and remove it:
+    if old_device := device_registry.async_get_device(
+        {(DOMAIN, system.system_id)}  # type: ignore[arg-type]
```

Comment:
`system_id` is an int, which used to be allowed as part of the device identifier tuple. To check for its existence, I need to disable `arg-type` type-checking.